utils/src/metrics.rs

```diff
@@ -484,7 +484,7 @@ pub fn export_files_stats(
             if !latest_read_files {
                 v.export_files_stats()
             } else {
-                v.export_latest_read_files()
+                Ok(v.export_latest_read_files())
```

Comment:
Clippy of version 1.51 or higher does not complain about this anymore.

Reply:
1.51.0 reports this error too, and even more Clippy errors, I think we can let this go and fix the others in a separate patch later.

Reply:
True.